builtin/game/detached_inventory.lua

```diff
@@ -15,6 +15,6 @@ function core.create_detached_inventory(name, callbacks)
 	end
 	stuff.mod_origin = core.get_current_modname() or "??"
 	core.detached_inventories[name] = stuff
-	return core.create_detached_inventory_raw(name)
+	return core.create_detached_inventory_raw(name, player_name)
```

Comment:
It can be hidden, it's also not documented in `lua_api.txt`.

However that's (obviously) not in the scope of my PR